matching-notification-api/src/main/java/org/phenotips/matchingnotification/storage/MatchStorageManager.java

```diff
@@ -49,7 +50,8 @@
      *        if {@code null}, then no upper limit on the match date is considered
      * @return a list of matches
      */
-    List<PatientMatch> loadMatches(double score, double phenScore, double genScore, boolean onlyCurrentUserAccessible,
+    List<PatientMatch> loadMatches(String patientId, double score, double phenScore, double genScore,
```

Comment:
I would add a separate call for loading matches for a given patient, with no score etc. etc. parameters, since we should always load all available matches for a patient. Internally they both can call the same private method with different parameters, if it makes life easier, but in API I would separate the two